packages/classic/src/core/challengePlayers.ts

```diff
@@ -39,7 +39,7 @@ export const getAll = zoddy(
     const players: z.infer<typeof playersSchema> = {};
 
     Object.entries(items).forEach(([username, data]) => {
-      players[username] = JSON.parse(data);
+      players[username] = JSON.parse(data) as (typeof players)['string'];
```

Comment:
I would define a type like
```ts
type PlayersSchema = z.infer<typeof playersSchema>;
```
And use it here instead
```suggestion
      players[username] = JSON.parse(data) as PlayersSchema;
```